python/cudf/cudf/utils/cudautils.py

```diff
@@ -17,33 +16,6 @@
     from numba import numpy_support
```

Comment:
I want to say that we now require a newer version of Numba anyway? But I may be mistaken

Reply:
You're right, we require 0.49 now. I'll cut this too.